### ISSUE TITLE
VGZ Support!

### ISSUE THREAD

**Issue by stevefuentes** [a-busman/SquareWaveSwift#1]
It'd be great if you would add VGZ support as well as VGM. Project 2612 has the full sega library all stored in vgz format, would be great to be able to play on the app!

**Comment by now-its-dark** [comment]
Yes please, I'd go a step further and say that it isn't really competitive without it at this point- project 2612 is the most comprehensive source of music for the mega drive.

**Comment by a-busman** [comment]
This should be simple to support, however, the only vgm I currently support is for the YM2612. This is good inspiration for using [libvgm](https://github.com/ValleyBell/libvgm). I'm currently working on SID support which includes a restructuring of audio emulation allowing for multiple different emulation libraries. This will be my next major addition for v2.0.
